pkg/infra/metrics/metrics.go

```diff
@@ -691,6 +695,12 @@ func init() {
 		Help:      "total amount of repositories",
 		Namespace: ExporterName,
 	})
+
+	MUnifiedStorageMigrationStatus = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
This metric will not expose any series until at least one reason is set. That may be OK, but I wonder if design with no `reason` label would work too? We could then define codes to have various meaning: `0 (default value) -- undefined`, `1 -- config disabled`, `2 -- would run`, etc.